torchao/quantization/GPTQ.py

```diff
@@ -648,6 +652,7 @@ def _create_quantized_state_dict(
                     weight,
                     4,  # n_bit
                     self.groupsize,
+                    self.precision, # precision for scales_and_zeros
```

Comment:
just terminology wise how do you seperate precision vs dtype

Reply:
Added a comment to explain precision == dtype. Keeping precision to match the other `Quantizer` classes.